src/main/kotlin/app/configuration/JobConfiguration.kt

```diff
@@ -18,7 +18,9 @@ import org.springframework.context.annotation.Bean
 import org.springframework.context.annotation.Configuration
 import org.springframework.context.annotation.Profile
 import org.springframework.core.task.SimpleAsyncTaskExecutor
+import org.springframework.core.task.TaskExecutor
 import org.springframework.retry.policy.SimpleRetryPolicy
+import uk.gov.dwp.dataworks.logging.DataworksLogger
```

Comment:
if this is new, can it be `DataWorksLogger`?

Reply:
Newish, we use it across the board,  logs everything in insight friendly json format.

Reply:
It's a month or so old and is in another repo.

Reply:
that's fine, if it was new today then worth changing, otherwise not